### PR TITLE
added to hub readme

### PR DESCRIPTION
**Pull request by harrysolovay**
*Description of changes:*

Added description of hub `tokenRefresh` and `tokenRefresh_failure` events. Also fixed accidental `error` instead of `info` for certain cases.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/utilities/fragments/hub.md [hunk 1]
```diff
@@ -105,19 +105,25 @@ const listener = (data) => {
     switch (data.payload.event) {
     
         case 'signIn':
-            logger.error('user signed in'); //[ERROR] My-Logger - user signed in
+            logger.info('user signed in'); //[ERROR] My-Logger - user signed in
```
Comment (sammartinez) [inline docs/lib/utilities/fragments/hub.md:108]: Can we adjust `//[ERROR]` if we are adjust this to an info?

**Review by sammartinez — COMMENTED** [review]
Small change  so once complete I can approve this.

File: docs/lib/utilities/fragments/hub.md (hunk 1 above)
Comment (harrysolovay) [reply docs/lib/utilities/fragments/hub.md:108]: What do you mean?

Comment (harrysolovay) [reply docs/lib/utilities/fragments/hub.md:108]: Ohhh, the comment. Yes.

**Review by sammartinez — APPROVED** [review]
LGTM 🌮

**Pull request merged**